src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -303,16 +267,15 @@ describe('NewWorkspaceModal', () => {
     // Assert
     // getByText throws an error if the element is not found:
     screen.getByText('Google Billing Project');
-    screen.getByText('Azure Billing Project');
     // queryByText returns null if the element is not found:
-    expect(screen.queryByText('Importing directly into new Azure workspaces is not currently supported.')).toBeNull();
+    expect(screen.queryByText('Importing directly into new Gcp workspaces is not currently supported.')).toBeNull();
```

Comment:
We do support import into GCP workspaces so I wouldn't expect this message to show up - we might want to keep the error message for Azure though?

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5275/commits/b69b154c73d52e08e5c9cdc1a4568356dde737d0